pallets/registrar/src/benchmarks.rs

```diff
@@ -375,7 +447,7 @@ mod benchmarks {
         }
 
         // Second loop to fill RegisteredParaIds to its maximum
-        for k in 1000..(1000 + y) {
+        for k in 1000..(1000 + y - 1) {
```

Comment:
is 1000 some sort of limit imposed by the runtime? if so can we fetch ir from there?